plugins/inputs/amqp_consumer/amqp_consumer.go

```diff
@@ -369,14 +370,19 @@ func (a *AMQPConsumer) declareQueue(channel *amqp.Channel) (*amqp.Queue, error)
 		queueDurable = false
 	}
 
+	queueArgs := make(amqp.Table, len(a.QueueArguments))
+		for k, v := range a.QueueArguments {
+			queueArgs[k] = v
+		}
```

Comment:
@n1b0r I think this should fix the linter issue
```suggestion
	for k, v := range a.QueueArguments {
		queueArgs[k] = v
	}
```